app/kumactl/pkg/cmd/root_context.go

```diff
@@ -241,3 +244,15 @@ func (rc *RootContext) cpOnTheSameMachine() (bool, error) {
 func (rc *RootContext) IsFirstTimeUsage() bool {
 	return rc.Args.ConfigFile == "" && !util_files.FileExists(config.DefaultConfigFile)
 }
+
+func (rc *RootContext) Version() (types.IndexResponse, error) {
+	controlPlane, err := rc.CurrentControlPlane()
+	if err != nil {
+		return types.IndexResponse{}, err
+	}
+	client, err := rc.Runtime.NewAPIServerClient(controlPlane.Coordinates.ApiServer.Url)
+	if err != nil {
+		return types.IndexResponse{}, errors.Wrap(err, "could not create components client")
+	}
+	return client.GetVersion()
```

Comment:
I somehow feel there shoudl be closer call for the `client` but it is not available. This way you leak resources IMO.